src/org/usfirst/frc/team467/robot/TalonSpeedControllerGroup.java

```diff
@@ -0,0 +1,192 @@
+package org.usfirst.frc.team467.robot;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.SpeedController;
+
+public class TalonSpeedControllerGroup implements SpeedController {
+	private static final Logger LOGGER = Logger.getLogger(TalonSpeedControllerGroup.class);
+	WPI_TalonSRX leader;
+	WPI_TalonSRX follower1;
+	WPI_TalonSRX follower2;
+	
+	private int previousSensorPosition;
+	
+	ControlMode controlMode = ControlMode.PercentOutput;
+	
+	public TalonSpeedControllerGroup() {
+		RobotMap.HAS_WHEELS = false;
+		leader = null;
+		follower1 = null;
+		follower2 = null;
+	}
+	
+	public TalonSpeedControllerGroup(ControlMode _controlMode, boolean sensorIsInverted,
+			WPI_TalonSRX _leader, WPI_TalonSRX _follower1, WPI_TalonSRX _follower2) {
```

Comment:
If you use underscores, you don't need the "this.foo"

Reply:
Don't use the underscore prefixes.